packages/terra-disclosure-manager/src/terra-dev-site/doc/disclosure-manager/DisclosureManagerAPI.3.doc.mdx

```diff
@@ -78,6 +78,12 @@ disclosureManager.disclose({
    *   > Note: `size` should not be provided if `dimensions` are specified.
    */
   size: PropsTypes.func,
+
+  /**
+   * Additional custom properties to override disclosed component.
+   * For instance to set panelBehavior when using a slidepanel manager use `typeConfig: { panelBehavior: 'squish' }`.
```

Comment:
Should we mention we only support panelBehavior at this point?

Reply:
We could. We can update once we start supporting other properties.

updated here a7145805d90a12ab96589eeb0474485d9e542aa3